recipes/dask-sphinx-theme/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.3" %}
+
+package:
+  name: dask-sphinx-theme
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/dask-sphinx-theme/dask_sphinx_theme-{{ version }}.tar.gz
+  sha256: ed4a56aea7f84707e0a87683463240b1ea249605f07c52b62c4c909cbbc20351
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - sphinx_rtd_theme
+
+test:
+  imports:
+    - dask_sphinx_theme
+
+about:
+  home: https://github.com/dask/dask-sphinx-theme
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Dask theme for Sphinx'
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - mrocklin
```

Comment:
@mrocklin, have gone ahead and added you here. Hope that is ok. If not, we can change this in the feedstock. Expect maintenance here to be pretty light.